amf-shapes/shared/src/main/scala/amf/shapes/internal/spec/oas/parser/InlineOasTypeParser.scala

```diff
@@ -99,7 +100,15 @@ case class InlineOasTypeParser(
   protected def isOas: Boolean  = version.isInstanceOf[OASSchemaVersion]
   protected def isOas3: Boolean = version.isInstanceOf[OAS30SchemaVersion]
 
-  def checkNilUnion(parsed: AnyShape): AnyShape = {
+  protected def moveExamplesToUnion(parsed: AnyShape, union: UnionShape): Unit = {
```

Comment:
Shouldn't this be in transformation ??

Reply:
not really, if the union is created in parsing it should be created correctly, not fixed in a later stage

Reply:
ahh because they are nullable fields.ok